src/css.js

```diff
@@ -113,6 +113,13 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableDimensionsInFullscreenMode && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		val = Math.round( elem.getBoundingClientRect()[ name ] * 100 );
```

Comment:
That'll give incorrect values, as `elem.offset*` will for instance be "3" where `getClientBoundingRect().width` would be 3.14 -- that's why I'm not using `offset*` here.